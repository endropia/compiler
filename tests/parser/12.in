type
 = Integer;
type
b = Char;
c = Real;
begin
end.