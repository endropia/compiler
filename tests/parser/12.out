(2, 2) Identifier expected